com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsRendering.cs

```diff
@@ -310,10 +363,10 @@ public bool TryGetScreenClearColor(ref Color color)
 
                 default:
                     throw new ArgumentOutOfRangeException(nameof(color));
-            }       // End of switch.
+            }
         }
 
-        public IDebugDisplaySettingsPanelDisposable CreatePanel()
+        IDebugDisplaySettingsPanelDisposable IDebugDisplaySettingsData.CreatePanel()
```

Comment:
Same question here regarding deprecating a public function...

Reply:
Same response here. Let me know if you agree!